yarn.lock

```diff
@@ -3104,10 +3104,10 @@
   resolved "https://registry.yarnpkg.com/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz#75a2e8b51cb758a7553d6804a5932d7aace75c39"
   integrity sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==
 
-"@box/cldr-data@^34.2.0":
-  version "34.2.0"
-  resolved "https://registry.yarnpkg.com/@box/cldr-data/-/cldr-data-34.2.0.tgz#d2a50e778d28988faaf5e56194b17b46c6b98ae5"
-  integrity sha1-0qUOd40omI+q9eVhlLF7Rsa5iuU=
+"@box/cldr-data@^34.5.0":
+  version "34.5.0"
```

Comment:
I know we're doing this in another PR but if this merges first we should include the BREAKING CHANGE piece here.

Reply:
I will change the commit msg

Reply:
Actually, already did that in commit becd6ab